pytest_fixtures/broker.py

```diff
@@ -9,6 +9,13 @@
 from robottelo.hosts import Satellite
 
 
+@pytest.fixture(scope='session')
+def default_sat():
+    """Returns a Satellite object for settings.server.hostname"""
+    if settings.server.hostname:
```

Comment:
I think eventually a lot of the logic from the align_xdist_worker fixture will move here, as they execute at the same scope.

Reply:
that could and maybe should be the case, since this fixture is going to become VERY important to the rest of the framework